DashAI/front/src/utils/index.js

```diff
@@ -1,6 +1,13 @@
-export const formatDate = (date) => {
-  if (date == null) {
+export const formatDate = (inputDate) => {
+  if (inputDate == null) {
     return "";
   }
-  return new Date(date).toLocaleDateString();
+  const date = new Date(inputDate);
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, "0");
+  const day = String(date.getDate()).padStart(2, "0");
+  const hours = String(date.getHours()).padStart(2, "0");
+  const minutes = String(date.getMinutes()).padStart(2, "0");
+
+  return `${year}/${month}/${day} ${hours}:${minutes}`;
```

Comment:
Maybe we should consider to use some js library to this ops?

- https://github.com/date-fns/date-fns
- https://github.com/iamkun/dayjs